Client/UIModalPresentationStyle+Photon.swift

```diff
@@ -0,0 +1,18 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0
+
+import UIKit
+
+extension UIModalPresentationStyle {
```

Comment:
The file path is at the root of client, I feel this file should be located elsewhere
![Screen Shot 2022-12-12 at 10 19 51 AM](https://user-images.githubusercontent.com/11338480/207082933-13b1ba8e-ad23-4efd-a5e8-e6fe799c13c8.png)